gpkit/tests/t_constraints.py

```diff
@@ -8,7 +8,8 @@
 from gpkit.constraints.loose import Loose
 from gpkit.tests.helpers import run_tests
 from gpkit.exceptions import InvalidGPConstraint
-from gpkit.constraints.relax import ConstraintsRelaxed
+from gpkit.constraints.relax import ConstraintsRelaxed, \
```

Comment:
instead of using slashes just add another line of import:

```python
from gpkit.constraints.relax import ConstraintsRelaxed
from gpkit.constraints.relax import ConstraintsRelaxedEqually, ConstantsRelaxed
```